styles/index.css

```diff
@@ -1,3 +1,62 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+}
+.overlay-text {
+  position: absolute;
+  color: rgb(234, 234, 233);
```

Comment:
Ideally we don't need to specify color at least for the header text. Can use 
```
color: white
opacity: 0.75
```
So it will blur nicely into the background